packages/kolibri-components/src/KThemePlugin.js

```diff
@@ -45,8 +45,30 @@ export default function KThemePlugin(Vue) {
       $inputModality() {
         return globalThemeState.inputModality;
       },
+      $mediaType() {
+        return globalThemeState.mediaType;
+      },
```

Comment:
could add

```javascript
$isPrint() {
  return this.$mediaType === 'print';
}
```

this would reduce the need for re-defining `isPrint` functions in all the components